packages/components/audio-player/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@yamada-ui/audio-player",
+  "version": "1.0.0",
+  "description": "Yamada UI AudioPlayer component",
+  "keywords": [
+    "yamada",
+    "yamada ui",
+    "react",
+    "emotion",
+    "component",
+    "accordion",
+    "ui",
+    "uikit",
+    "styled",
+    "style-props",
+    "styled-component",
+    "css-in-js"
+  ],
+  "author": "Hirotomo Yamada <hirotomo.yamada@avap.co.jp>",
+  "license": "MIT",
+  "main": "src/index.ts",
+  "files": [
+    "dist"
+  ],
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://yamada-ui.com",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yamada-ui/yamada-ui",
+    "directory": "packages/components/audio-player"
+  },
+  "bugs": {
+    "url": "https://github.com/yamada-ui/yamada-ui/issues"
+  },
+  "scripts": {
+    "dev": "pnpm build:fast -- --watch",
+    "build": "tsup src --dts",
+    "build:fast": "tsup src",
+    "clean": "rimraf dist .turbo",
+    "typecheck": "tsc --noEmit",
+    "prepack": "clean-package",
+    "postpack": "clean-package restore",
+    "gen:docs": "tsx ../../../scripts/generate-docs"
+  },
+  "dependencies": {
+    "@yamada-ui/button": "workspace:^",
+    "@yamada-ui/core": "workspace:*",
+    "@yamada-ui/slider": "workspace:^",
+    "@yamada-ui/utils": "workspace:*"
```

Comment:
Do not use yamada-ui components such as `Button` or `Slider`.
(`@yamada-ui/core` and `@yamada-ui/utils` are fine.)